orchagent/portsorch.cpp

```diff
@@ -2128,6 +2173,44 @@ task_process_status PortsOrch::setPortAutoNeg(sai_object_id_t id, int an)
     return task_success;
 }
 
+bool PortsOrch::getPortCapabilityLinkTraining(const Port& port)
+{
+    string supp;
+
+    if (m_stateLinkTrainingTable.hget(port.m_alias, "supported", supp))
```

Comment:
Please use internal Port data structure to cache the capability rather than writing to DB and reading from it within the same process

Reply:
This is now cached as member field of the internal Port data structure